src/test/kotlin/graphics/scenery/tests/examples/advanced/EyeTrackingExample.kt

```diff
@@ -20,9 +17,11 @@ import kotlin.concurrent.thread
  * @author Ulrik Günther <hello@ulrik.is>
  */
 class EyeTrackingExample: SceneryBase("Eye Tracking Example", windowWidth = 1280, windowHeight = 720) {
-    val pupilTracker = PupilEyeTracker(calibrationType = PupilEyeTracker.CalibrationType.ScreenSpace)
+    val pupilTracker = PupilEyeTracker(calibrationType = PupilEyeTracker.CalibrationType.WorldSpace)
```

Comment:
When the constructor has default parameters, the calibrationType could be left out here